src/app/content/highlights/components/Confirmation.tsx

```diff
@@ -17,15 +17,14 @@ export const Overlay = styled.div`
   outline: none;
   ${cardBorder}
   transition: background 200ms;
-  position: absolute;
   display: flex;
   flex-direction: column;
   padding: 1.6rem;
-  top: 0;
-  left: 0;
   overflow: visible;
   min-height: 100%;
   min-width: 100%;
+  position: static;
+  width: min-content;
```

Comment:
The login confirmation would spread out when on narrower screens. This fixes that.